cloud/services/networkinterfaces/service.go

```diff
@@ -43,11 +43,11 @@ type Service struct {
 	InternalLoadBalancersClient internalloadbalancers.Client
 	PublicIPsClient             publicips.Client
 	InboundNATRulesClient       inboundnatrules.Client
-	ResourceSkusClient          resourceskus.Client
+	ResourceSkusCache           *resourceskus.Cache
```

Comment:
is ResourceSkusCache needed as part of the Service struct? Isn't it already part of Scope?

Reply:
whoops -- good catch. I had it flipped before.

Reply:
actually, because we use the constrained interface here, we can't pluck it from the scope

Reply:
I think that's fine though, I'd rather be explicit passing it down